### PR TITLE
Adding some custom special field options

### PR DESCRIPTION
**Pull request by Boolunt**
Doesn't fix the issue but I they're additions towards solving #9203
Apologies for the large amount of commits.

Quality
Relevance

- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by tobiasdiez** [comment]
Nice addition, thanks for your work!

I was wondering though what is the advantage of having the options "Relevant/Not relevant" over the existing toggle option "Relevance". If we would want to "expand" the relevance option (instead of making it a simple yes/no), then maybe it make sense to directly introduce a 3 or 5 option grading?

I know you tried to be consistent with the other options, but in my opinion the "Set quality to low" is too verbose and doesn't add anything. I would propose to have the following options (in this order) under "Quality":
- High
- Medium
- Low
- Clear (that this is a special option is communicated by the absence of the colored quality icon, but maybe add a menu separator before it)

Not sure what the other @JabRef/developers think about this though.

**Comment by Boolunt** [comment]
Hi sorry for the late response. I'm able to keep working on this now though. I agree with all your suggestions and am happy to make those changes.

File: src/test/java/org/jabref/migrations/SpecialFieldsToSeparateFieldsTest.java [hunk 1]
```diff
@@ -20,7 +20,7 @@ class SpecialFieldsToSeparateFieldsTest {
     @ParameterizedTest
     @MethodSource("provideKeywordFieldPairs")
     public void migrateToCorrectField(SpecialField field, String fieldInKeyword, BibEntry expected) {
-        BibEntry entry = new BibEntry().withField(StandardField.KEYWORDS, fieldInKeyword);
+        BibEntry entry = new BibEntry().withField(field, fieldInKeyword);
```
Comment (koppor) [inline src/test/java/org/jabref/migrations/SpecialFieldsToSeparateFieldsTest.java:23]: The intention of this test class is to check migration of special field values in the `keywords` field. Thus, please to not alter this test.

In the past, JabRef offered storing special fields in keywords (see https://github.com/JabRef/jabref/blob/main/docs/decisions/0019-implement-special-fields-as-separate-fields.md#special-fields-as-keywords). We decided to move away from that.

File: src/test/java/org/jabref/migrations/SpecialFieldsToSeparateFieldsTest.java [hunk 2]
```diff
@@ -87,7 +87,7 @@ private static Stream<Arguments> provideKeywordFieldPairs() {
                         SpecialField.PRIORITY, "prio1", new BibEntry().withField(SpecialField.PRIORITY, "prio1")
                 ),
                 Arguments.of(
-                        SpecialField.QUALITY, "qualityAssured", new BibEntry().withField(SpecialField.QUALITY, "qualityAssured")
+                        SpecialField.QUALITY, "quality1", new BibEntry().withField(SpecialField.QUALITY, "quality1")
```
Comment (koppor) [inline src/test/java/org/jabref/migrations/SpecialFieldsToSeparateFieldsTest.java:90]: Altering test cases should be done with care. Why is the old field `qualityAssuered` removed? What about backwards compatibility?

**Review by koppor — COMMENTED** [review]
Just first minor comments (for now). Do you intend to continue on this PR?

**Comment by calixtus** [comment]
Closing this PR due to inactivity :zzz:

**Pull request closed without merging**